### ISSUE TITLE
"No log data" for pods streaming out json

### ISSUE THREAD

**Issue by andrewnazarov** [IBM/kui#4782]
**Describe the bug**
For some of our pods we are constantly getting  "No log data" under the Logs tab. It seems that this happens for pods which stream out logs in json format. For plain text logs, everything seems to work nicely.

**Steps to reproduce the behavior**

1. You need a pod with json logs output
2. 'kubectl get po'
3. Click on the pod
4. Go to the "Logs" tab

**Expected behavior**
Logs for all pods no matter what is the format of logs output

**Screenshots**
<img width="767" alt="Screenshot 2020-05-31 at 15 24 19" src="https://user-images.githubusercontent.com/11255710/83352333-60363600-a353-11ea-920d-ec51e8a585bc.png">

**System**

- Kui version: 8.7.6
- Operating system: macOS Mojave 10.14.5

**Comment by starpit** [comment]
thanks for the bug report. we tried this yaml, and it seemed to work. it's possible the bug has been fixed in 8.8.0. we haven't yet released builds, but should within the next 24h. with master, this yaml works, using the test script you provided:

```yaml
apiVersion: v1
kind: Pod
metadata:
  name: json-logs
spec:
  containers:
  - image: alpine
    command:
      - /bin/sh
      - "-c"
      - "while true; do echo '{\"key\": \"value\"}'; sleep 1; done"
    name: alpine
```

**Comment by starpit** [comment]
ah yes, this should be fixed in 8.8.0. we improved the robustness of the Logs tab quite a bit. Illl ping this issue when we have released 8.8.0!

**Comment by starpit** [comment]
8.8.2 has been released.
